Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -228,15 +228,15 @@ public class Klaviyo : NSObject {
         let customerPropertiesDict = updatePropertiesDictionary(propDictionary: customerProperties)
         assertPropertyTypes(properties: propertiesDict)
         
+        guard let apiKey = apiKey else {
+            environment.logger.error("Track event called before API key was set.")
```

Comment:
should we replace existing usages of `print` with this logger too?

Reply:
Eventually yah...trying to keep this focused though. I can file a ticket for it.